BoostTestAdapter/Boost/Runner/BoostTestRunnerBase.cs

```diff
@@ -151,11 +151,11 @@ private static void KillProcessIncludingChildren(Process process)
             // Killing the main process
             if (KillProcess(process))
             {
-                Logger.Error("Successfully killed process {0}.", process.Id);
+                Logger.Error(Resources.KilledProcess, process.Id);
```

Comment:
Could we here and below use "terminate process" verbiage instead?

Reply:
Resolved